ratatool-scalacheck/README.md

```diff
@@ -39,7 +37,45 @@ Implicit Arbitrary instances are also available for Avro and Protobuf records. E
 
 ```scala
 val avroArb: Arbitrary[MyRecord] = implicitly[Arbitrary[MyRecord]]
+```
+
+Multiple `Gen` instances can be composed together with a for-comprehension, which will cause the data to be generated with the same seed,
```

Comment:
Gonna nitpick more sorry, the text reads as if composing gens always results in failure. we should move the failure text down to sample, and adjust it to say something like
```
To extract a single value from a generator you can just `.sample`, however this can result in failures in cases where the seed is unable to produce a valid value. See `withGen` etc etc etc`

Reply:
Also can you add a header to this section to separate the `amend()` stuff

Reply:
Reworded.